src/main/java/org/xerial/snappy/OSInfo.java

```diff
@@ -226,6 +231,16 @@ public static String getArchName() {
         return translateArchNameToFolderName(osArch);
     }
 
+    private static boolean isMusl() {
+        try {
+            return new File("/lib/ld-musl-x86_64.so.1").exists();
+        } 
+        catch (Exception e) {
+            System.err.println("Error while checking if ld-musl-x86_64.so.1 exists: " + e.getMessage());
```

Comment:
Generally speaking, consuming error is not a good idea, but showing an error here could be annoying for other OS/JVM/security configurations, which might not allow access to the /lib folder. 

So remove this error message.